src/app/home/home.component.ts

```diff
@@ -49,6 +52,15 @@ export class HomeComponent {
         });
       });
     }
+
+    this.wsService.onMessage()
```

Comment:
This would be easier to read:
```ts
this.wsService.onMessage()
  .filter(() => this.currentTab !== 'chat')
  .subscribe(() => this.newMessages++);
```